grow/pods/collection.py

```diff
@@ -263,8 +263,11 @@ def list_categories(self):
     def list_docs(self, order_by=None, locale=utils.SENTINEL, reverse=None,
                   include_hidden=False, recursive=True, inject=False):
         reverse = False if reverse is None else reverse
-        order_by = 'order' if order_by is None else order_by
-        key = operator.attrgetter(order_by)
+        if order_by is None:
+            order_by = ('order', 'pod_path')
```

Comment:
assuming that `pod_path` is the full path relative to the pod root, the collection path prefix will always be the same, so might be better to order by the `doc.base`, which would have less chars for the str compare.

Reply:
It does, but I believe that collections can have sub directories that are part of the collection as well? If so then the basename would mess up the sorting between the sub directories if we just use the base.